### DISCUSSION TITLE
display-mode: standalone

### DISCUSSION THREAD

**Discussion by tadinski** [chakra-ui/chakra-ui#4960] [Q&A]
How would one go about creating a media query for [standalone display-mode](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/display-mode).

Currently using sx prop on the component but I would like to create a separate media query.

I would expect something like this to work but I have no luck getting it implemented:

`const breakpoints = createBreakpoints({
 standalone: "@media screen and (display-mode: standalone)"
})`

**Comment by primos63** [comment]
Did you try [useMediaQuery](https://chakra-ui.com/docs/hooks/use-media-query)?

```jsx
function Example() {
  const [isStandalone] = useMediaQuery("(@media screen and (display-mode: standalone))")

  return (
    <Text>
      {isStandalone ? "Standalone" : "Not Standalone"}
    </Text>
  )
}
```

**Reply by tadinski** [reply]
thanks for the suggestion @primos63 

I am able to get the media query recognised in standalone mode without` @media screen` like this:
useMediaQuery("(display-mode: standalone)") 

However, I get an error

<img width="1112" alt="Screenshot 2021-11-05 at 14 15 46" src="https://user-images.githubusercontent.com/6358371/140516048-b3d1c8e2-8557-4969-8d3c-506516b5aca0.png">
 

I believe it has something to do with SSR as I am using Nextjs.

**Comment by tadinski** [comment]
solution to my own problem can be found here:

https://github.com/chakra-ui/chakra-ui/issues/4319#issuecomment-878237998

**Reply by primos63** [reply]
Ah so implementing a `useEffect` for so that the `useMediaQuery` is only run when your app is hydrated is the SSR solution. Good to know!

**Reply by tadinski** [reply]
that's right!
